<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>578</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>578</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>FinWise</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(19, 27, 38)</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>811</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>811</width>
      <height>581</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
border: 1px solid gray;
}

QTabWidget::tab-bar {
alignment: center;
}

QTabBar::tab {
padding: 6px 10px;
margin: 0px 5px;
border: none;
color: white;
}

QTabBar::tab:selected {
background-color: rgb(33, 44, 58);
border-bottom: 1px solid rgb(45, 117, 251);
padding-top: 7px;
}

QTabBar::tab:hover {
background-color: rgb(35, 46, 60);
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="homeTab">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>611</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ffffff;&quot;&gt;FinWise - Financial Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logTransactionButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3b82f6;
color: #ffffff;
border: none;
border-radius: 8px;
padding: 5px 10px;
font-weight: 600;
}

QPushButton:hover {
background-color: #2563eb;
}

QPushButton:pressed {
background-color: #1d4ed8;
}</string>
      </property>
      <property name="text">
       <string>Log Transaction</string>
      </property>
     </widget>
     <widget class="QLabel" name="subtextLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#bebebe;&quot;&gt;What would you like to do today?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="accountBalanceMain">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="accountBalanceMainLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your account balance is:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transactionsTab">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Transactions</string>
     </attribute>
     <widget class="QPushButton" name="updateTransactionButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #1e3a8a;
color: #93c5fd;
border: 1px solid #3b82f6;
border-radius: 8px;
padding: 5px 10px;
}

QPushButton:hover {
background-color: #1e40af;
color: #ffffff;
border-color: #60a5fa;
}</string>
      </property>
      <property name="text">
       <string>Update Selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteTransactionButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #7f1d1d;
color: #fca5a5;
border: 1px solid #dc2626;
border-radius: 8px;
padding: 5px 10px;
}

QPushButton:hover {
background-color: #991b1b;
color: #ffffff;
border-color: #ef4444;
}</string>
      </property>
      <property name="text">
       <string>Delete Selected</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="transactionsTable">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>50</y>
        <width>751</width>
        <height>480</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
background-color: rgb(33, 44, 58);
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="supportedDragActions">
       <set>Qt::DropAction::IgnoreAction</set>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="background">
        <color>
         <red>33</red>
         <green>44</green>
         <blue>58</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="sortLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort by:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortByCategoryDropdown">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>10</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Amount</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="sortingOrderDropdown">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>10</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>Descending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ascending</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="statisticsTab">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="chartSelectDropdown">
        <item>
         <property name="text">
          <string>Spending by Category</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Income by Category</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Income vs Expense</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monthly Trend</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="chartContainer" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="profileTab">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QGroupBox" name="xpInformation">
        <property name="title">
         <string>Level Information</string>
        </property>
        <widget class="QLabel" name="accountXpLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;XP Points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountXp">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="basicInformation">
        <property name="title">
         <string>Basic Information</string>
        </property>
        <widget class="QLabel" name="accountNameLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountName">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountBalanceLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>68</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountBalance">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountTransactionsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Transactions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountTransactions">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>90</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="settings">
        <property name="title">
         <string>Preferences</string>
        </property>
        <widget class="QLabel" name="categoriesInputLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Categories&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="categoriesInput">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>601</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Enter categories</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveSettingsButton">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>220</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #3b82f6;
color: #ffffff;
border: none;
border-radius: 8px;
padding: 5px 10px;
font-weight: 600;
}

QPushButton:hover {
background-color: #2563eb;
}

QPushButton:pressed {
background-color: #1d4ed8;
}</string>
         </property>
         <property name="text">
          <string>Save Changes</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountNameInputLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Account Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="accountNameInput">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>201</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Enter name</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionAdd_Transaction">
   <property name="text">
    <string>Add Transaction</string>
   </property>
  </action>
  <action name="actionView_Transactions">
   <property name="text">
    <string>View Transactions</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionXP_Level">
   <property name="text">
    <string>XP Level</string>
   </property>
  </action>
  <action name="actionWeekly">
   <property name="text">
    <string>Weekly</string>
   </property>
  </action>
  <action name="actionMonthly">
   <property name="text">
    <string>Monthly</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
